### DISCUSSION TITLE
External library not scanned

### DISCUSSION THREAD

**Discussion by enboig** [immich-app/immich#7829] [Q&A]
Using immich in docker. I followed https://immich.app/docs/guides/external-library/ and added one path:
```
services:
  immich-server:
    volumes:
      - ${EXTERNAL_PATH1}:/usr/src/app/external/external1
```
and `.env`:
```
EXTERNAL_PATH1=/mnt/md0/Imatges
```
I entered the container to check everything:
```
# docker exec -it immich_server bash
oot@fff32ab6f920:/usr/src/app# ls /usr/src/app/external/external1/*.jpg | head 
/usr/src/app/external/external1/colmena.jpg
/usr/src/app/external/external1/scan0001.jpg
```
But library finds 0 assets.

Any idea?

**Comment by bo0tzz** [comment]
Have you actually applied the changes with `docker compose up -d`?

**Reply by enboig** [reply]
yes, they are. If folder shows when entering the container, volume is mounted and files accessible (I also tried `cat` them to check read access

**Reply by bo0tzz** [reply]
Can you post a screenshot of the external library config as well?

**Reply by enboig** [reply]
![imatge](https://github.com/immich-app/immich/assets/1155203/166e4061-ee19-449a-9bea-835eeaa08eed)

**Reply by enboig** [reply]
![imatge](https://github.com/immich-app/immich/assets/1155203/bdc07602-12d9-4e46-9e9f-4933b673cf2f)

**Reply by enboig** [reply]
I used external1 just in case I want to add more external path to docker volumes later.

**Reply by enboig** [reply]
I forgot to include external1 volume to `immich_microservices`, trying again and waiting.

**Reply by enboig** [reply]
Scan started. Thanks for your time.
